ImmuDB4Net.Tests/BasicClientIntegrationTests.cs

```diff
@@ -152,6 +152,27 @@ public async Task TestSimpleVerifiedSetAndGet()
         TxHeader hdr1 = await client.VerifiedSet("key1", "value1");
         Assert.IsNotNull(hdr1);
 
+        Entry ventry1 = await client.VerifiedGet("key1");
+        Assert.AreEqual("value1", ventry1.ToString());
+        await client.Close();
+    }    
+    
+    [TestMethod("execute expirableset and verifiedget")]
+    public async Task TestExpirableSet()
+    {
+        await client!.Open("immudb", "immudb", "defaultdb");
+
+        byte[] v0 = new byte[] { 0, 1, 2, 3 };
+        byte[] v1 = new byte[] { 3, 2, 1, 0 };
+
+        TxHeader hdr0 = await client.ExpirableSet("k0", v0, DateTime.Now.AddDays(1));

```

Comment:
Can we also add a test when the key expires? like:

```go
TxHeader hdr0 = await client.ExpirableSet("k0", v0, DateTime.Now.AddDays(-1));
```

Reading such entry should end up with `key not found` error.

Reply:
yes, done.